config/routes.rb

```diff
@@ -115,7 +115,7 @@
       resource :moving_requirements, only: %i[new create]
       resource :applicable_exemptions, only: %i[new create]
       # resource :check_your_answers, only: %i[new create]
-      resource :results, only: %i[show]
+      resources :results, param: :category, only: %i[show]
```

Comment:
Is this a query/positional param constraint? Not sure we need constraints for routes rendered/managed by us?

Reply:
This allows us to use:

'/green_lanes/results/1' in the routes
and get the params['category'] == '1'  

for example
---
Not sure about your point, let's chat about it